praw/__init__.py

```diff
@@ -1173,7 +1173,7 @@ def _url_update(self, url):
                 url = url.replace("user/"+redditor, 'me')
         return url
 
-    @decorators.restrict_access(scope=None, login=True)
+    @decorators.restrict_access(scope='modself', login=True)
```

Comment:
I went with what the original documentation says here: https://www.reddit.com/dev/api/oauth#scope_modself and apparently here: https://github.com/reddit/reddit/blob/master/r2/r2/controllers/api.py#L1160

Do what you want with it, my only actual problem was with modqueue; accepting moderation isn't very common for my scripts (but it could be *very* common for bots such as automoderator, I guess)